docs/src/main/sphinx/release/release-412.md

```diff
@@ -0,0 +1,100 @@
+# Release 412 (5 Apr 2023)
+
+## General
+
+* Add support for aggregate functions and parameters as arguments for the
+  [`json_object()`](json_object) and [`json_array()`](json_array)
+  functions. ({issue}`16489`, {issue}`16523`, {issue}`16525`)
+* Expose optimizer rule execution statistics in query statistics. The number of
+  rules statistics are collected for can be limited with the
+  `query.reported-rule-stats-limit` configuration property. ({issue}`2578`)
+* Add the {func}`exclude_columns` table function. ({issue}`16584`)
+* Allow disabling the use of the cost-based optimizer to determine partitioning
+  of a stage with the `optimizer.use-cost-based-partitioning`configuration
+  property or the `use_cost_based_partitioning` session property. ({issue}`16781`)
+* Improve performance of queries involving table functions with table arguments. ({issue}`16012`)
+* Improve latency for small queries when fault-tolerant execution is enabled. ({issue}`16103`)
+* Fix failure when querying a nested field of a `row` type in queries involving
+  `ORDER BY ... LIMIT`. ({issue}`16768`)
+
+## JDBC driver
+
+* Allow configuring a custom DNS resolver. ({issue}`16647`)
+
+## ClickHouse connector
+
+* Improve performance of queries involving `sum(distinct)` by pushing
+  computation of it to ClickHouse. ({issue}`16452`)
+
+## Delta Lake connector
+
+* Add support for arithmetic binary expressions in table check constraints. ({issue}`16721`)
```

Comment:
```suggestion
* Add support for arithmetic binary expressions in table ``CHECK`` constraints. ({issue}`16721`)
```

check is a syntax element.

Reply:
Either way is ok. In the SQL spec they are called "table check constraints"